Section 5 - Remote State Management/terraform-multistate/ec2/ec2.tf

```diff
@@ -1,8 +1,8 @@
 
 resource "aws_instance" "myec2" {
```

Comment:
**Tenable.cs remediation**
 
**[LOW]: Ensure there are no unnamed AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances without a name may cause confusion during resource audit and maintenance.
 
**Remediation**
In AWS Console - 
 1. Sign in to AWS Console and go to the EC2 dashboard. 
 2. Select the region.
 3. In the navigation pane, choose a resource type.
 4. Under Add Tag, enter the tag key and value and choose Add Tag.
 5. Click Save.

source: tenablecs-